source/SkiaSharp.Views.Forms/SkiaSharp.Views.Forms.Shared/SKTouchEventArgs.cs

```diff
@@ -42,9 +42,11 @@ public SKTouchEventArgs(long id, SKTouchAction type, SKMouseButton mouseButton,
 
 		public int WheelDelta { get; private set; }
 
+		public float Pressure { get; internal set; } = 1.0f;
```

Comment:
Maybe add an overload for this in the constructor so this args can be created from outside SkiaSharp.

Reply:
will do